README.md

```diff
@@ -449,6 +449,7 @@ Config:
     	    - KoF     ( 1440x1920 ) - Kobo Forma
     	    - KoS     ( 1440x1920 ) - Kobo Sage
     	    - KoE     ( 1404x1872 ) - Kobo Elipsa
+    	    - RM      ( 1404x1872 ) - reMarkable 2
```

Comment:
can you set RM2 here ?

usually I do
```
go run . -help
```
then I copy paste it in the readme
and to ensure I don't put my default, I do a 
```
go run . -reset
```

But you may want to save your config so you can do something like
```
mv ~/.go-comic-converter.yaml ~/.go-comic-converter.yaml.old
go run . -help
mv ~/.go-comic-converter.yaml.old ~/.go-comic-converter.yaml
```